ui/perfherder/compare/CompareTableView.jsx

```diff
@@ -154,6 +166,27 @@ export default class CompareTableView extends React.Component {
     this.setState({ timeRange }, () => this.getPerformanceData());
   };
 
+  notify = (message, severity, options = {}) => {
```

Comment:
I think we should DRY this and `clearNotification()` also, maybe in a helper module.

Reply:
I think we should find a way to combine `failureMessage` with notifications. In other Perfherder views, all error messages are added to one array and they are displayed at the top of the page in the parent component. Looking at this pr, after a job is retriggered it shows a message that is aligned to the left of the page. So rather than try to mimic notify that's used in Treeherder, you can use [ErrorMessages](https://github.com/mozilla/treeherder/blob/cee89fb9e935d966f786e57a780f2f88b33c6b99/ui/perfherder/compare/CompareTableView.jsx#L275) and refactor it to just 'messages' or 'notifications' that could pass in a color prop (or you could create a new component based on that). This would keep it consistent with how messages are shown in all Perfherder views.

Reply:
And for the sake of simplicity, I'd just show a message that the retrigger succeeded or failed rather than the blue 'it's been retriggered' message since we don't do that anywhere else in Perfherder.